x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_snooze/panel/base_snooze_panel.tsx

```diff
@@ -76,6 +76,7 @@ export const BaseSnoozePanel: React.FunctionComponent<BaseSnoozePanelProps> = ({
   const snoozeRuleAndStoreInterval = useCallback(
     (newSnoozeEndTime: string | -1, intervalToStore: string | null) => {
       if (intervalToStore) {
+        if (intervalToStore.startsWith('-')) throw new Error('Cannot store a negative interval');
```

Comment:
Just thinking of other edge cases, but would it be more robust if we only allowed positive integers? (using regex or something).

I don't have any strong feelings but could be worth considering.

Reply:
Not sure what a more robust regex would be other than making sure the string doesn't start with a `-` is? This takes strings in the format of `{integer}{timeUnit}` like `3h`, and we're trying to prevent an input like `-3h`. Open to a suggestion on what would be a more robust check.

Reply:
ah I see, we're already getting the interval at this point. 

but yea I was thinking what if the user were to input `03h`, or decimals, but yea we can just address them as they come.